docs/install-kyma-dashboard-manually.md

```diff
@@ -35,3 +35,24 @@ To run Kyma Dashboard manually on a cluster, you must install it together with I
 
 3. Wait a minute for the Dashboard to start.
 4. The Dashboard is available under your domain (for example, `https://busola.local.kyma.dev/`).
+
+## Enable TLS in the backend service
+
+By default, the communication with the backend pod is plain HTTP. Even when you use istio service mesh, the last bit of communication
+between the istio sidecar and the backend service stays plain HTTP.
+
+If your use-case requires TLS all the way into the backend pod, you can enable it by providing a TLS certificate to the backend service,
+setting some environment variables and changing the deployment.
+
+1. Generate TLS certificate
```

Comment:
```suggestion
1. Generate a TLS certificate.
```

Is any TLS certificate good, or does it have to fulfil any special requirements regarding encoding etc?